### ISSUE TITLE
Number of instances displayed vs written disagree

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#2042]
Like in this figure  
<img width="247" alt="Screen Shot 2021-11-09 at 15 35 58" src="https://user-images.githubusercontent.com/7316439/140944064-c526dd00-5cdb-4117-a0ec-559e2859a49a.png">  
which was generated for a paper.

We see 10 steps in the ECDF, but `15 instances` written. Why?

See also #1972, #1117.

**Comment by brockho** [comment]
After quite some experimentation, I found the cause: The number of steps in the ECDFs is not the number of instances, but the number of bootstrapped runs. Because line 50 of `genericsettings.py`:
```
simulated_runlength_bootstrap_sample_size = 10 + 990 // (1 + 10 * max((0, in_a_hurry)))  # for tables and plots
```
evaluates to 10 with the default setting of `in_a_hurry`, we see only 10 (simulated) runs in the figure. Changing line 50 to, for example, `7` and with `pprldmany_target_values=[1e-3]` in `testbedsettings.py` gives the following output with the `ALPS` data set:

![image](https://user-images.githubusercontent.com/13706996/140962372-740b2cee-a2a0-46dd-aaef-221f2cefacdb.png)

**Comment by brockho** [comment]
Update: I also found the same line
`settings.simulated_runlength_bootstrap_sample_size = 10 + 990 / (1 + 10 * max(0, settings.in_a_hurry))`
within `config.py`...

**Comment by nikohansen** [comment]
Excellent, thanks, that means we should probably better use the maximum of the number of trials and number of bootstraps. We still need to fix #1972 at some point, I believe I have already some code around.

**Comment by nikohansen** [comment]
Should be fixed in the `development` branch.
